### ISSUE TITLE
accessibility issue: shortcut UI not accessible via screen reader

### ISSUE THREAD

**Issue by RichCaloggero** [notepad-plus-plus/notepad-plus-plus#11832]
### Description of the Issue

The shortcut UI (run menu \ modify shortcut command) shows some controls like filter, modify, and clear. However, it also contains a control simply announced by the screen reader as "pane". I cannot access this control at all, and I suspect this is where the shortcut definitions are displayed and can be modified.

Because I cannot see, I can't provide any more information here. The UI is completely inaccessible, so I have no suggestions on how it might be fixed.

### Steps to Reproduce the Issue

1. click "run" from top menubar
2. click "modify shortcut" from menu that appears

### Expected Behavior

There should be a set of standard controls here one can use to show a list of commands / macros / whatever, and a way to associate a shortcut key sequence with each item in the list.

### Actual Behavior

The control which presumably allows one to see and modify shortcuts is not accessible via screen reader.

### Debug Information

No add-ons, plugins, etc are being used. this is straight out of the box.

**Comment by alankilborn** [comment]
Duplicate of #6060
